sources/web/datalab/reverseProxy.ts

```diff
@@ -48,12 +48,22 @@ export function getRequestPort(request: http.ServerRequest, path: string): strin
 }
 
 /**
+ * Checks if a request should be exempted from reverse proxying to the gateway
+ */
+function shouldSkipGateway(port: String) {
```

Comment:
I agree the naming isn't good. I think it's a better idea to make the function check the KG_URL though, but it should be checking if traffic should be proxied, not local. Please take a look and let me know what you think.